Azure/azure-cli
# Sql

**Description**

Breaking change in SQL API.  readReplicaCount was renamed to highAvailabilityReplicaCount

**Testing Guide**

Existing tests exercise this and required update to verify renamed property


---

This checklist is used to make sure that common guidelines for a pull request are followed.
